### ISSUE TITLE
false positive "declarations and declarations with annotations should have an empty space between."

### ISSUE THREAD

**Issue by hgschmie** [pinterest/ktlint#1281]
## Expected Behavior
A Kotlin class with asciidoc tag references gets parsed correctly and no error is reported.

## Observed Behavior
erroneous messages about 'Declarations and declarations with annotations should have an empty space between.' are reported

## Steps to Reproduce
This is a kotlin file with asciidoc references: https://github.com/jdbi/jdbi/blob/master/docs/src/test/kotlin/jdbi/doc/KotlinPluginTest.kt

Running it through "ktlint --experimental` results in:

```
/Users/henning/code/jdbi/docs/src/test/kotlin/jdbi/doc/KotlinPluginTest.kt:79:5: Declarations and declarations with annotations should have an empty space between.
/Users/henning/code/jdbi/docs/src/test/kotlin/jdbi/doc/KotlinPluginTest.kt:88:5: Declarations and declarations with annotations should have an empty space between.
```

It is not clear to my why this is reported. It seems that ktlint is trying to parse the contents of the line comments (which are used by asciidoc to reference part of the source code in the documentation). 

## Your Environment

* Version of ktlint used: maven-plugin 1.11.0, using ktlint 0.43
* Maven 3.8.3
* Java version: 17.0.1, vendor: Homebrew, runtime: /usr/local/Cellar/openjdk/17.0.1/libexec/openjdk.jdk/Contents/Home
* Default locale: en_US, platform encoding: UTF-8
* OS name: "mac os x", version: "11.6.1", arch: "x86_64", family: "mac"

* Link to your project (if it's a public repository): https://github.com/jdbi/jdbi/
